samples/crypto/aes_ccm/prj.conf

```diff
@@ -16,4 +16,5 @@ CONFIG_MBEDTLS_ENABLE_HEAP=y
 CONFIG_MBEDTLS_HEAP_SIZE=8192
```

Comment:
Hi, using this branch I built this sample and it built fine.

But to my surprise disabling CONFIG_PSA_WANT_KEY_TYPE_AES did not produce a build-time error.

IMHO we cannot introduce this option without validating it's correctness at build time.

Reply:
Will be fixed in Georgios's PR.